### ISSUE TITLE
Cross-account security-group references broken

### ISSUE THREAD

**Issue by seren** [dtan4/terraforming#390]
Hi, terraforming is producing security-group ingress rules which have invalid cross-account references:
```
resource "aws_security_group" "vpc-abcd1234-classic-vpc-linkage" {
    name        = "classic-vpc-linkage"
    description = "classic - vpc - internal traffic"
    vpc_id      = "vpc-abcd1234"

    ingress {
        from_port       = 1234
        to_port         = 1234
        protocol        = "udp"
        security_groups = ["832806758429/", "832806758429/", "832806758429/"]
        self            = true
    }
}
```

This occurs on both Linux using the latest docker container, as well as on MacOS using ruby 2.4.0 and Terraforming version 0.15.0.

The cause is that the `group_name` for a permission's `user_id_group_pair` may be `nil` when doing cross-account references. A solution is to fall back to using the `group_id` in these cases. I've created a PR for this, using the fallback solution: #389 

Let me know if there are any problems with this solution, and thanks for your time!

**Comment by seren** [comment]
Since this PR has been merged, would it be possible to push an updated gem version? It would make building docker images a bit easier. Thanks!

**Comment by asrivastava-github** [comment]
Yeah, please push the changes to gem version please.

**Comment by dtan4** [comment]
Sorry for the late. I released this change as v0.17.0. Thanks!
